meta/main.yml

```diff
@@ -10,6 +10,7 @@ galaxy_info:
       versions:
         - jessie  # 8
         - stretch # 9
+        - buster  # 10
     - name: Ubuntu
       versions:
         - trusty  # 14.04
```

Comment:
Perhaps we should remove this as we don't have this version in the molecule configuration but most important IMHO as with the other comment for Debian 8, Ubuntu Trusty reached its EOL in April 2019. Not sur if we still need to support it. What do you think @sestrella and @nickovivar ? :)

Reply:
@speedlight good catch! We should probably get rid of Trusty since it is not even listed on `molecule` `platforms` section.